selene_sdk/samplers/multi_file_sampler.py

```diff
@@ -1,231 +0,0 @@
-"""
```

Comment:
this file is still deleted in the branch - can you replace it and add deprecation warning upon initialization (and in the documentation?)

Reply:
currently if people import `selene_sdk.samplers.MultiFileSampler` it will still work and provides a deprecation warning (MultiFileSampler is moved to multi_sampler.py), so we want people to be also able to import selene_sdk.samplers.multi_file_sampler.MultiFileSampler right?